ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/activity/PaywallActivityLauncher.kt

```diff
@@ -73,9 +73,9 @@ class PaywallActivityLauncher {
      */
     @JvmOverloads
     fun launchIfNeeded(
+        requiredEntitlementIdentifier: String,
         offering: Offering? = null,
         fontProvider: ParcelizableFontProvider? = null,
-        requiredEntitlementIdentifier: String,
```

Comment:
Honestly I wasn't sure about doing this now, but I think it's better now that the API is still experimental. If we don't put this mandatory parameter first, users are forced to use named parameters if they only want to set the `requiredEntitlementIdentifier` but not the optional parameters.

Reply:
Yes, better now than never! I don't think there are that many users yet so let's fix these things while we have time.